backend/LfClassicData/LfClassicMiniLcmApi.cs

```diff
@@ -157,7 +164,7 @@ private async IAsyncEnumerable<Entry> Query(QueryOptions? options = null, string
 
         await foreach (var entry in Entries.Aggregate(pipeline).ToAsyncEnumerable())
         {
-            yield return ToEntry(entry);
+            yield return await ToEntry(entry); // TODO: Is there a way to turn this into a .Select()? Because "yield return await" looks a little unwieldy
```

Comment:
yes, but we would still need to use the `await foreach` so it would still be ugly, just somewhere else.

Reply:
Comment removed in commit 295c23ee.